static/css/devhub/base.less

```diff
@@ -1,32 +1,45 @@
 @import '../common/normalize.css';
 @import '../impala/fonts/fira/fira.less';
 
-// Colors
-@primary-font: 'Fira Sans', sans-serif;
-@primary-font-color: #fff;
-@secondary-font-color: #333;
-@tertiary-font-color: #40d38d;
-
-@button-border: #979797;
-@primary-button-background: #29c978;
-
-@primary-section-background: #0568ba;
-@secondary-section-background: #f3f5f8;
-@tertiary-section-background: #1d3770;
-
+// Grid
+@grid-max: 940px;
 @phone-large: ~"only screen and (min-width: 428px)";
 @phone: ~"only screen and (max-width: 768px)";
```

Comment:
Not sure what you mean by separating things into chunks. 

We'd generally be looking to just override the mobile styling (default since mobile first) on a case by case basis and those media queries should be used each time next to the thing it's overriding as opposed to separating the file out into blocks relating to each MQ (is that what you meant by chunks).

I'd also look at avoiding overlaps with the media-queries unless absolutely needed for some special cases.